.gitignore

```diff
@@ -1 +1,4 @@
-/data
\ No newline at end of file
+/data
+.vscode
+app
+cmd/ddns-updater/data/
```

Comment:
Revert this 🙏  (or do another PR if you want to add i.e. `.vscode` to the .gitignore)

Reply:
#835 handled here